### PR TITLE
Update zh-TW

### PR DESCRIPTION
**Pull request by KuwaLee**
Please help me review @auroursa

### CODE REVIEW

**Review by auroursa — COMMENTED** [review]
As I mentioned before, I'm not sure if running intl:extract manually is necessary
Other than that, LGTM

File: src/locale/locales/zh-TW/messages.po [hunk 1]
```diff
@@ -942,30 +914,30 @@ msgstr "為 {name} 配置內容過濾設定"
 
 #: src/components/moderation/LabelPreference.tsx:244
 msgid "Configured in <0>moderation settings</0>."
-msgstr "已在<0>限制設定</0>中配置。"
+msgstr "已在 <0> 內容管理設定 </0> 中配置。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:917]:
```suggestion
msgstr "已在 <0> 內容管理設定 </0> 中配置。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 2]
```diff
@@ -1140,15 +1108,15 @@ msgstr "無法載入列表"
 
 #: src/components/dms/NewChat.tsx:241
 msgid "Could not load profiles. Please try again later."
-msgstr ""
+msgstr "無法載入設定檔。請稍後再試。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:1111]:
```suggestion
msgstr "無法載入個人資料。請稍後再試。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 3]
```diff
@@ -2058,35 +2031,35 @@ msgstr "前往首頁"
 msgid "Go to next"
 msgstr "前往下一步"
 
-#: src/components/dms/ConvoMenu.tsx:114
+#: src/components/dms/ConvoMenu.tsx:133
 msgid "Go to profile"
-msgstr ""
+msgstr "轉跳至個人頁面"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2036]:
```suggestion
msgstr "跳轉至個人頁面"
```

File: src/locale/locales/zh-TW/messages.po [hunk 4]
```diff
@@ -2058,35 +2031,35 @@ msgstr "前往首頁"
...
 
-#: src/components/dms/ConvoMenu.tsx:111
+#: src/components/dms/ConvoMenu.tsx:130
 msgid "Go to user's profile"
-msgstr ""
+msgstr "轉跳至用戶個人頁面"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2040]:
```suggestion
msgstr "跳轉至用戶個人頁面"
```

File: src/locale/locales/zh-TW/messages.po [hunk 5]
```diff
@@ -2331,7 +2308,7 @@ msgstr "邀請碼：{0} 個可用"
 
 #: src/view/com/modals/InviteCodes.tsx:170
 msgid "Invite codes: 1 available"
-msgstr "邀請碼：1 個可用"
+msgstr "邀請碼：1個可用"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2311]:
```suggestion
msgstr "邀請碼：1 個可用"
```

建議數字後留空，格式看起來比較統一

File: src/locale/locales/zh-TW/messages.po [hunk 6]
```diff
@@ -2481,23 +2446,9 @@ msgstr "按喜歡的用戶"
...
-#: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:287
-#: src/screens/Profile/Header/ProfileHeaderLabeler.tsx:301
-#: src/view/screens/ProfileFeed.tsx:600
-#~ msgid "Liked by {likeCount} {0}"
-#~ msgstr "{likeCount} 個 {0} 喜歡"
-
 #: src/view/com/notifications/FeedItem.tsx:168
 msgid "liked your custom feed"
-msgstr "已按喜歡您的自訂動態流"
+msgstr "已按喜歡您的自訂動態源"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2451]:
```suggestion
msgstr "對您的自訂動態源表示喜歡"
```

File: src/locale/locales/zh-TW/messages.po [hunk 7]
```diff
@@ -2624,99 +2580,99 @@ msgstr "被提及的用戶"
...
 #: src/screens/Moderation/index.tsx:215
 msgid "Moderation tools"
-msgstr "限制工具"
+msgstr "內容管理工具"
 
 #: src/components/moderation/ModerationDetailsDialog.tsx:48
 #: src/lib/moderation/useModerationCauseDescription.ts:40
 msgid "Moderator has chosen to set a general warning on the content."
-msgstr "限制者已選擇對內容設定普通警告。"
+msgstr "內容管理者已選擇對內容設定普通警告。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2675]:
```suggestion
msgstr "內容管理者已將此內容標記為普通警告。"
```

File: src/locale/locales/zh-TW/messages.po [hunk 8]
```diff
@@ -2974,26 +2930,26 @@ msgstr "關"
...
 
-#: src/view/com/modals/ChangeHandle.tsx:406
+#: src/view/com/modals/ChangeHandle.tsx:399
 msgid "No DNS Panel"
 msgstr "無 DNS 控制台"
 
 #: src/components/dialogs/GifSelect.tsx:206
 msgid "No featured GIFs found. There may be an issue with Tenor."
-msgstr "未找到精選 GIF，Tenor 可能發生問題。"
+msgstr "未找到精選GIF，Tenor可能發生問題。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2939]:
```suggestion
msgstr "未找到精選 GIF，Tenor 可能發生問題。"
```

建議英文前後留空，格式看起來比較統一

File: src/locale/locales/zh-TW/messages.po [hunk 9]
```diff
@@ -2974,26 +2930,26 @@ msgstr "關"
...
 
 #: src/screens/Profile/Header/ProfileHeaderStandard.tsx:116
 msgid "No longer following {0}"
 msgstr "不再跟隨 {0}"
 
 #: src/screens/Signup/StepHandle.tsx:115
 msgid "No longer than 253 characters"
-msgstr "不超過 253 個字符"
+msgstr "不超過253個字符"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2947]:
```suggestion
msgstr "不超過 253 個字符"
```

建議數字前後留空，格式看起來比較統一

File: src/locale/locales/zh-TW/messages.po [hunk 10]
```diff
@@ -3016,15 +2972,15 @@ msgstr "未找到「{query}」的結果"
 #: src/view/screens/Search/Search.tsx:289
 #: src/view/screens/Search/Search.tsx:328
 msgid "No results found for {query}"
-msgstr "未找到 {query} 的結果"
+msgstr "未找到{query}的結果"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2975]:
```suggestion
msgstr "未找到 {query} 的結果"
```

File: src/locale/locales/zh-TW/messages.po [hunk 11]
```diff
@@ -3016,15 +2972,15 @@ msgstr "未找到「{query}」的結果"
...
 
 #: src/components/dialogs/GifSelect.tsx:204
 msgid "No search results found for \"{search}\"."
 msgstr "未找到「{search}」的搜尋結果。"
 
 #: src/components/dms/NewChat.tsx:240
 msgid "No search results found for \"{searchText}\"."
-msgstr ""
+msgstr "未找到以下搜尋結果「{searchText}」"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:2983]:
```suggestion
msgstr "未找到與「{searchText}」相關的搜尋結果"
```

File: src/locale/locales/zh-TW/messages.po [hunk 12]
```diff
@@ -4199,7 +4161,7 @@ msgstr "從下面選擇您感興趣的選項"
 
 #: src/view/screens/LanguageSettings.tsx:190
 msgid "Select your preferred language for translations in your feed."
-msgstr "選擇您在動態源中翻譯的偏好的目標語言。"
+msgstr "選擇您在動態中翻譯的偏好的目標語言。"
```
Comment (cirx1e) [inline src/locale/locales/zh-TW/messages.po:4164]:
```suggestion
msgstr "選擇您在動態中翻譯的偏好目標語言。"
```

**Review by cirx1e — COMMENTED** [review]
修正了一些用詞

File: src/locale/locales/zh-TW/messages.po (hunk 2 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:1111]: This is when search people that failed to load search profile so it is inappropriate to use "個人",  but thanks i will chanege word

File: src/locale/locales/zh-TW/messages.po (hunk 3 above)
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:2036]: I prefer "轉跳", but "前往" can be use also

File: src/locale/locales/zh-TW/messages.po [hunk 13]
```diff
@@ -5436,44 +5380,40 @@ msgstr "驗證我的電子郵件"
...
 
-#: src/components/ReportDialog/SelectReportOptionView.tsx:132
+#: src/components/ReportDialog/SelectReportOptionView.tsx:138
 msgid "View details"
 msgstr "查看詳細資訊"
 
-#: src/components/ReportDialog/SelectReportOptionView.tsx:127
+#: src/components/ReportDialog/SelectReportOptionView.tsx:133
 msgid "View details for reporting a copyright violation"
 msgstr "查看詳細資訊以檢舉侵權"
```
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:5418]: You are right!!!!

File: src/locale/locales/zh-TW/messages.po [hunk 14]
```diff
@@ -5749,14 +5697,14 @@ msgstr "您已靜音這個用戶"
 
 #: src/view/com/feeds/ProfileFeedgens.tsx:144
 msgid "You have no feeds."
-msgstr "您沒有訂閱動態源。"
+msgstr "您沒有建立任何動態源。"
```
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:5700]: This is when you go to your profile there is no feed create by you

File: src/locale/locales/zh-TW/messages.po [hunk 15]
```diff
@@ -5749,14 +5697,14 @@ msgstr "您已靜音這個用戶"
...
 
 #: src/view/com/lists/MyLists.tsx:89
 #: src/view/com/lists/ProfileLists.tsx:148
 msgid "You have no lists."
-msgstr "您沒有列表。"
+msgstr "您沒有建立任何列表。"
```
[the remark this answers is not in the document]
Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:5705]: This is when you go to your profile there is no list create by you

**Comment by cirx1e** [comment]
> It's a bit awkward because I just made changes here... If 「使用者」 is more commonly used, I'll roll back the changes. 
The previous translation mixed 「用戶」 and 「使用者」, so this time I just wanted to make the translation more consistent.

_Originally posted in https://github.com/bluesky-social/social-app/issues/3678#issuecomment-2078908649_

---

接續之前的討論，臺灣應是「使用者」更常用？
沒有異議的話，建議把「用戶」統一替換成「使用者」

@KuwaLee 你的看法？

## Twitter
Screenshot_20240512_010000_X
>  Source: Twitter 帳戶設定


## Mastodon
bb9fc66e-0ec1-4627-92d8-da8ceecc9cef
> Source: Mastodon 註冊頁面


## Google
835f8d37-ec26-4143-b0d8-03a2785798ea
> Source: Google 建立帳號頁面

**Comment by KuwaLee** [comment]
@cirx1e 之前討論過在這份翻譯中用戶指有帳之user，沒有帳號的user是使用者

File: src/locale/locales/zh-TW/messages.po [hunk 16]
```diff
@@ -1,6 +1,6 @@
 msgid ""
```
Comment (a681120g) [inline src/locale/locales/zh-TW/messages.po:1]: src/locale/locales/zh-TW/messages.po

Comment (KuwaLee) [reply src/locale/locales/zh-TW/messages.po:1]: ？？？？

**Comment by pfrazee** [comment]
How's this looking? Good to merge?

**Pull request closed without merging**